samples/cinema_video.json

```diff
@@ -0,0 +1,8 @@
+{
+    "_id": "4e1a31e14b032f2fa9e161ee9b009125",
+    "dc_title": "Video Buster Keaton",
+    "dc_creator": "Buster Keaton",
+    "dc_issued": "2023-02-21",
+    "text": " [![](https://www.profession-spectacle.com/wp-content/uploads/2016/01/Buster-Keaton.jpg)](https://www.youtube.com/watch?v=JRXkAhMYKEc)",
+    "dc_license": "All rights reserved"
```

Comment:
You don't have to put "All rights reserved" as a license. This is the default display when no license is set.